src/schema.jl

```diff
@@ -322,11 +322,13 @@ the type of experiment (`"QASM"` or `"PULSE"`) and/or the type of backend (e.g.
 simulator data). See below.
 """
 @option struct ExpResult <: IBMQSchema
-    metadata::Maybe{Dict{String, Any}} = nothing
-    header::Maybe{Dict{String, Any}} = nothing
-    shots::Union{Int, Vector{Int}}
+    metadata::Maybe{Dict{String,Any}} = nothing
+    header::Maybe{Dict{String,Any}} = nothing
+    shots::Union{Int,Vector{Int}}
     status::String
     success::Bool
+    meas_level::Int
+    circ_id::Int
     time_taken::Float64
```

Comment:
just to understand the change better cuz there are a lot format changes, is this the main update of the result schema?

Reply:
Yes, correct. Sorry for the formatting...